```
error[E0308]: mismatched types
   --> /Users/servo/buildbot/slave/mac-dev-unit/build/components/gfx/font_cache_thread.rs:352:90
    |
352 |                     (None, Some(native_font)) => webrender_api.add_native_font(font_key, native_font),
    |                                                                                          ^^^^^^^^^^^ expected struct `webrender_traits::NativeFontHandle`, found struct `core_graphics::font::CGFont`
    |
    = note: expected type `webrender_traits::NativeFontHandle`
               found type `core_graphics::font::CGFont`

error: aborting due to previous error
```
Want to pull https://github.com/servo/servo/pull/16512 into this as well?

--- a/components/gfx/platform/macos/font_template.rs
+++ b/components/gfx/platform/macos/font_template.rs
@@ -18,6 +18,7 @@ use std::fs::File;
 use std::io::{Read, Error as IoError};
 use std::ops::Deref;
 use std::sync::Mutex;
+use webrender_traits::NativeFontHandle;
 
 /// Platform specific font representation for mac.
 /// The identifier is a PostScript font name. The
@@ -106,8 +107,8 @@ impl FontTemplateData {
     }
 
     /// Returns the native font that underlies this font template, if applicable.
-    pub fn native_font(&self) -> Option<CGFont> {
-        self.ctfont(0.0).map(|ctfont| ctfont.copy_to_CGFont())
+    pub fn native_font(&self) -> Option<NativeFontHandle> {
+        self.ctfont(0.0).map(|ctfont| NativeFontHandle(ctfont.copy_to_CGFont()))
     }
 }
 

--- a/tests/wpt/metadata-css/css-transforms-1_dev/html/transform-input-018.htm.ini
+++ b/tests/wpt/metadata-css/css-transforms-1_dev/html/transform-input-018.htm.ini
@@ -1,3 +1,5 @@
 [transform-input-018.htm]
   type: reftest
-  expected: FAIL
+  expected:
+    if os == "linux": FAIL
+

